src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const priceRentDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+
+  let totalAmount = days * priceRentDay;
```

Comment:
Consider using `const` instead of `let` for `totalAmount` since it's not being reassigned after its initial calculation. If you need to adjust the value based on the discounts, you can return the adjusted value directly within the `if` blocks.